### ISSUE TITLE
[Feature Request] Emoji Picker

### ISSUE THREAD

**Issue by CSab6482** [open-imcore/imcore.react#16]
My main reason for this is that my system emojis don't have support for Unicode 13, and it also autocorrects some emojis to emoticons. I personally like [Emoji Mart](https://github.com/missive/emoji-mart) (also [here](https://missive.github.io/emoji-mart/)), which I'm pretty sure is what WebMessage and BlueBubbles Desktop use, but I'll ask to make sure.

**Comment by CSab6482** [comment]
https://github.com/iandwelker/smserver/issues/115

**Comment by zlshames** [comment]
Yeah, that's what BlueBubbles Server uses. The library is super weird.... but it works

**Comment by CSab6482** [comment]
https://github.com/BlueBubblesApp/BlueBubbles-Desktop-App/issues/193#issuecomment-816248349
